civihr_employee_portal/src/Forms/OnboardingWizardCustomizationForm.php

```diff
@@ -177,6 +185,18 @@ private function clickedCancel($formState) {
     return \CRM_Utils_Array::value('#value', $clickedButton) === t('Cancel');
   }
 
+  /**
+   * On form submission checks if the button pressed was "SaveAndTest"
+   *
+   * @param array $formState
+   *
+   * @return bool
+   */
+  private function clickedSaveAndTest($formState) {
+    $clickedButton = \CRM_Utils_Array::value('clicked_button', $formState);
```

Comment:
I think we usually have an empty line before `return`

Reply:
Ok, I will add that line.